samples/rest/BingVisualSearchV7.py

```diff
@@ -1,46 +1,49 @@
-# Download and install Python at https://www.python.org/ 
+# Download and install Python at https://www.python.org/
 # Run the following in a command console window: pip3 install requests
```

Comment:
the comment in line 2 is a bit outdated, it should rely on the requirements.txt file

Reply:
Done e9ef3aa